addons/website_sale/models/sale_order.py

```diff
@@ -161,7 +160,7 @@ def _cart_update(self, product_id=None, line_id=None, add_qty=0, set_qty=0, **kw
             set_qty = 0
         quantity = 0
         order_line = False
-        if self.state != 'draft':
+        if self.state not in ('draft', 'sent'):
```

Comment:
The 'm' commit does not change the fact after checkout you still retrieve the sent order when doing sale_get_order.
More important it allows now to work on the 'sent' order after a checkout so you can modify the previous order see https://drive.google.com/file/d/1MkpMRy4qweNyJ2aPAjXVQl0kKtT1Nrjo/view